tcms/testruns/views.py

```diff
@@ -280,9 +281,8 @@ class EditTestRunView(UpdateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context['test_plan'] = self.object.plan
+        context['plan'] = self.object.plan
         context['builds'] = Build.objects.filter(product=self.object.plan.product, is_active=True)
```

Comment:
`plan` is not used in mutable.html as a context variable, everything else is coming from the form and it shouldn't be necessary. 

Looks to me that `builds` wasn't used as well.

Reply:
Then you have: 290         form.populate(self.object.plan.product_id)
which uses BaseRunForm but that won't filter form.plan.queriset appropriately IMO and the data shown inside the HTML template relies on that. So change the form class and adjust the call to populate.

Reply:
I didnt get the second part, nevertheless I did some testing and compare the form in edit and new(with same plan) and it worked as expected.